src/components/PostList.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Post from './Post';
+
+const PostList = props => props.posts.map(post => (
+  <Post post={post} />
```

Comment:
Три слова post подряд не очень читается
```suggestion
  <Post post={postData} />
```